blocks/author-list/author-list.js

```diff
@@ -0,0 +1,133 @@
+import ffetch from '../../scripts/ffetch.js';
+import { readBlockConfig, loadBlocks, createOptimizedPicture } from '../../scripts/lib-franklin.js';
+
+/**
+ * loads and decorates the footer
+ * @param {Element} block The footer block element
+ */
+export default async function decorate(block) {
+  const { filter } = readBlockConfig(block);
+  block.textContent = '';
+  // eslint-disable-next-line no-console
+  fetchAuthors(filter, block).catch((e) => console.log(e));
+}
+
+async function fetchAuthors(filter, block) {
+  // get all authors from authors.json and filter them by role
+  const authors = await ffetch('/authors.json').filter((author) => author.role === filter).all();
+  // sort author list by name
+  authors.sort((a, b) => a.name.localeCompare(b.name));
+  // create the first 6 authors
+  const numInitialLodedAuthors = 6;
+  const firstAuthors = authors.slice(0, numInitialLodedAuthors);
+  firstAuthors.forEach((author) => {
+    const newBlock = createAuthorCardBlock(author);
+    block.append(newBlock);
+  });
+  // create load more button if there are more authors than shown
+  if (authors.length > numInitialLodedAuthors) {
+    const { loadMoreButton, loadMoreContainer } = createLoadMoreButton();
+    loadMoreButton.addEventListener('click', () => {
+      // get the next 50 authors
+      const nextAuthors = authors.slice(numInitialLodedAuthors);
+      nextAuthors.forEach((author) => {
+        const newBlock = createAuthorCardBlock(author);
+        block.append(newBlock);
+      });
+      loadMoreContainer.remove();
+    });
+    block.append(loadMoreContainer);
+  }
+  loadBlocks(block);
+}
+
+function createLoadMoreButton() {
+  const loadMoreContainer = document.createElement('div');
+  loadMoreContainer.classList.add('author-load-more-container');
+  const loadMoreButton = document.createElement('button');
+  loadMoreContainer.append(loadMoreButton);
+  loadMoreButton.classList.add('author-list-load-more-button');
+  loadMoreButton.textContent = 'Load more';
+  return { loadMoreButton, loadMoreContainer };
+}
+
+function buildAuthorListItem(className, content) {
+  const list = Array.isArray(content) ? content : [content];
```

Comment:
This function can probably be simplified. Think a div.append([…content]) should do most of this

Reply:
fixed